### DISCUSSION TITLE
full time sandwich menu

### DISCUSSION THREAD

**Discussion by douglasjsmith404** [importantimport/urara#77] [General]
wondering if the main menu can remain a 'triple-line' compressed menu even at full width.
or can i configure menu expanded but vertical like on the 'docs' site.

[my gh pages blog](https://douglasjsmith404.github.io)
[repo](https://github.com/douglasjsmith404/douglasjsmith404.github.io)

**Comment by Interstellar750** [comment]
What you want should be this drop down menu:

<img width="468" src="https://github.com/importantimport/urara/assets/38517305/32a03b66-1e59-471c-988b-ae8316d23bb5" />

The setting method of this drop-down menu in `src/lib/config/general.ts` is:
```ts
export const header: HeaderConfig = {
  nav: [
    {
      text: "Drop-down menu title",
      children: [
        {
          text: "First url",
          link: "/url"
        },
        {
          text: "Second url",
          link: "/url2"
        },
        {
          text: "Third url",
          link: "/url3"
        }
      ]
    },
    {
      text: "Other url",
      link: "/other_url"
    }
  ]
}
```

Of course you will get a style error, this is a problem since [00dca49](https://github.com/importantimport/urara/commit/00dca49042ce38bc8194aec7cb498e8fe72e6013), there is no fix yet, you can fall back to the previous [6748aa4](https://github.com/importantimport/urara/commit/6748aa450af8efd73a5cd62d86ab5150b698113c) to temporarily avoid this problem

Or you can follow the [3f91e9c](https://github.com/Interstellar750/hexo_s/commit/3f91e9c4ceec52fa37d6c74dc403d7ee710c774f) commit in my repository to temporarily fix this error, but it is not perfect, you need adjust the `class="w-36"` style I added, and the drop-down menu will not be aligned with the button in the header, in mobile layout, all drop-down menu will be expanded and can't be close.

**Comment by douglasjsmith404** [comment]
thanks,
  following your commit and dropdown menu is retracting as it should.  did have to move 'coding' to lowest row of main items as well to make it function correctly. lower row items interfered  

https://douglasjsmith404.github.io
